packages/airnode-admin/test/e2e/cli.feature.ts

```diff
@@ -660,4 +660,24 @@ describe('CLI', () => {
       expect(out).toEqual(`Airnode address: ${airnodeWallet.address}`);
     });
   });
+
+  describe('has valid examples', () => {
+    const examples = [
+      'derive-airnode-address --airnode-mnemonic "test test test test test test test test test test test junk"',
+      'derive-sponsor-wallet-address --airnode-xpub xpub6Ce9NcJvTk36xtLSrJLZqE7wtgA5deCeYs7rSQtreh4cj6ByPtrg9sD7V2FNFLPnf8heNP3FGkeV9qwfzvZNSd54JoNXVsXFYSYwHsnJxqP --airnode-address 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 --sponsor-address 0x70997970c51812dc3a010c7d01b50e0d17dc79c8',
+      'derive-endpoint-id --ois-title "title" --endpoint-name "endpoint"',
+    ];
```

Comment:
It looks like these examples cannot be exported from `cli.ts` (maybe because it's the `yargs` command file? and it ended up running the commands instead) so I just copied in the array, but these could be moved to a common `cli-examples.ts` file if that's better.

Reply:
Yeah, that would be better.